styles/components/_button.scss

```diff
@@ -1,9 +1,18 @@
 // stub, should be fleshed out
 .cmp-button {
-  padding: 0.25rem 1rem;
+  padding: space("sm");
+  padding-right: 0;
   font-size: font-size("xs");
 
   &--link {
-    text-decoration: underline;
+    @media (hover: hover) {
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    &:focus {
+      @include focus-styles;
+    }
```

Comment:
This isn't the right SCSS file for this, but the Copy Bio button's lopsided.
<img width="468" alt="Screen Shot 2022-06-14 at 2 06 49 PM" src="https://user-images.githubusercontent.com/53844657/173679244-25af30ad-23ec-4c8a-a8bf-cf0ad1dd65e3.png">
Adding `padding: 0;` to `.cmp-copy-to-clipboard-button` fixed this for me.